tests/testthat/test_SmoothnessL1.R

```diff
@@ -0,0 +1,54 @@
+context("Smoothness")
+
+test_that("SmoothnessL1 defines a norm", {
+    order <- 5L
+
+    design0 <- gq_design(25, 0, 2, rep(0.0, order), rep(2, order), order)
+
+
+    design1 <- gq_design(25, 0, 2, rep(40.0, order), rep(2, order), order)
+
+    design2 <- design1
+    design2@n2_pivots <- 2 * design1@n2_pivots
+
+    design3 <- design0
+    design3@n2_pivots <- design1@n2_pivots + design2@n2_pivots
+
+    smth <- SmoothnessL1()
+
+    expect_equal(
+        evaluate(smth, design0),
+        0.0
+    )
+
+    expect_equal(
+        evaluate(smth, design2),
+        2 * evaluate(smth, design1)
+    )
+
+    expect_equal( # everything is positive!
+        evaluate(smth, design3),
+        evaluate(smth, design1) + evaluate(smth, design2)
+    )
+}) # end 'SmoothnessL1 defines a norm'
+
```

Comment:
i think we have 3 blanks between test_that - makes it easier to switch cases quickly